packages/rest/src/router/routing-table.ts

```diff
@@ -150,6 +157,9 @@ export class RoutingTable {
 
     validateApiPath(route.path);
     this._router.add(route);
+    if (route.path === '*') {
```

Comment:
This is a bit hacky to test `path === '*'`. The special route is also added to the underlying router, which might cause unexpected results.